timelock-server/src/main/java/com/palantir/atlasdb/timelock/config/TimeLockServerConfiguration.java

```diff
@@ -139,8 +139,8 @@ private int computeNumberOfAvailableThreads() {
         Preconditions.checkState(serverFactory.getApplicationConnectors().get(0) instanceof HttpConnectorFactory,
                 "applicationConnectors of TimeLockServerConfiguration must have a HttpConnectorFactory instance.");
         HttpConnectorFactory connectorFactory = (HttpConnectorFactory) serverFactory.getApplicationConnectors().get(0);
-        int selectorThreads = connectorFactory.getSelectorThreads();
-        int acceptorThreads = connectorFactory.getAcceptorThreads();
+        int selectorThreads = connectorFactory.getSelectorThreads().orElse(0);
```

Comment:
I believe the minimum is 1 for each of these, don't think it matters too much here, but would be good to consistent

Reply:
Where is that defined?

Reply:
`HttpConnectorFactory`

Reply:
Sorry, I meant where is it stated that the minimum is 1

Reply:
https://github.com/dropwizard/dropwizard/blob/master/dropwizard-jetty/src/main/java/io/dropwizard/jetty/HttpConnectorFactory.java#L283